lib/hca/enrollment_eligibility/status_matcher.rb

```diff
@@ -131,29 +131,35 @@ def process_text_match(text_matches, ineligibility_reason)
         nil
       end
 
+      # rubocop:disable Metrics/CyclomaticComplexity
+      # rubocop:disable Metrics/PerceivedComplexity
```

Comment:
If you're interested, I saw this recently.  I believe you can bring these into one line like this:

```ruby
 # rubocop:disable Metrics/PerceivedComplexity, Metrics/CyclomaticComplexity
```